app/src/org/commcare/views/media/CommCareVideoView.java

```diff
@@ -39,6 +39,10 @@ public void start() {
     @Override
     protected void onDetachedFromWindow() {
         super.onDetachedFromWindow();
+        // Do not log events if the video is never played.
+        if (duration == 0) {
```

Comment:
Can we always fire the listener and do this check in the listener implementation. Would also be good to separate this out from this PR.

Reply:
Separated out this change here https://github.com/dimagi/commcare-android/pull/2179